cdap-tms/src/main/java/io/cdap/cdap/messaging/client/DelegatingMessagingService.java

```diff
@@ -130,6 +137,11 @@ private MessagingService getDelegate() {
             "Unsupported messaging service implementation " + getName());
       }
       LOG.info("Messaging service {} is loaded", messagingService.getName());
+      try {
+        messagingService.initialize(new DefaultMessagingServiceContext(this.cConf));
```

Comment:
Given that this may throw an IO exception, you should also log a message for `Messaging service {} is initialized` after successful initialization

Reply:
Done